unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeLoaderController.cs

```diff
@@ -145,11 +148,11 @@ public void UpdateBackgroundColor(Color newColor)
     private void FetchNFTContents()
     {
         ShowLoading(true);
-        nftInfoLoadHelper.FetchNFTInfo(darURLRegistry, darURLAsset, FetchNFTInfoSuccess, FetchNFTInfoFail);
+        nftInfoLoadHelper.FetchNFTInfo(darURLRegistry, darURLAsset);
```

Comment:
I was thinking what if we do many requests one after another? But I think it would work fine, they would just use the same callback at the end anyways and be affected by the unsubscription.